plugins/vsphere/rockcraft.yaml

```diff
@@ -0,0 +1,86 @@
+# Dockerfile: https://github.com/vmware-tanzu/velero-plugin-for-vsphere/blob/v1.5.2/Dockerfile-plugin
+name: velero-plugin-for-vsphere
+summary: Velero Plugin for vSphere rock
+description: |
+    A rock containing the Velero Plugin for vSphere.
+
+    Velero gives you tools to back up and restore your Kubernetes cluster
+    resources and persistent volumes.
+license: Apache-2.0
+version: 1.5.2
+
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+environment:
+  APP_VERSION: 1.5.2
+  LD_LIBRARY_PATH: /plugins
+
+# Services to be loaded by the Pebble entrypoint
+services:
+  install-plugin:
+    override: replace
+    startup: enabled
+    # Added a sleep to the command to prevent it from immediately exiting.
+    # This way, Pebble respects the on-success / on-failure rules.
```

Comment:
That there is different Pebble behavior between >1s and <1s commands amazes me ...